package.json

```diff
@@ -63,7 +63,7 @@
     "lint:lockfile:dedupe:fix": "yarn dedupe",
     "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn github.com codeload.github.com --empty-hostname true --allowed-schemes \"https:\" \"git+https:\" \"npm:\" \"patch:\" \"workspace:\"",
     "lint:shellcheck": "./development/shellcheck.sh",
-    "lint:styles": "stylelint -- */**/*.scss",
```

Comment:
Is this change necessary because lint wasn't running at all on Windows?

Reply:
See the second bullet point in the PR description: "on Windows, produced the error `The command line is too long.`"